package.json

```diff
@@ -45,7 +45,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js icons"
   },
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Comment:
@ClementChaumel let's drop the version change, we can release it together with what's in main branch